app/docker-compose.prod.yml

```diff
@@ -60,6 +60,8 @@ services:
     networks:
       - media_nginx
       - envoy_nginx
+  cron:
+    build: cron
```

Comment:
If this is a separate container, it needs to be built in CI/CD, hopefully tested, then the prod docker-compose should pull the image from gitlab, it shouldn't be built in prod.